1.10/Dockerfile

```diff
@@ -60,6 +60,9 @@ RUN sed -i -f ${NGINX_APP_ROOT}/nginxconf.sed /etc/opt/rh/rh-nginx110/nginx/ngin
     mkdir -p ${NGINX_APP_ROOT}/etc/nginx.default.d/ && \
     mkdir -p ${NGINX_APP_ROOT}/src/nginx-start/ && \
     mkdir -p ${NGINX_CONTAINER_SCRIPTS_PATH}/nginx-start && \
+    mkdir -p /var/opt/rh/rh-nginx110/logs/nginx && \
+    ln -sf /dev/stdout /var/opt/rh/rh-nginx110/log/nginx/access.log && \
```

Comment:
Should this line (and the previous/next) ref `NGINX_SHORT_VER` instead of hard-coding the version in `.../rh-nginx110/...`?

Reply:
I changed it to use that var, and then extended it to several other lines... *but* there are several i didn't touch.  Feels like it more should be converted? whats the distinction between doing it or not?